pkl-core/src/test/files/LanguageSnippetTests/input/api/bytes.pkl

```diff
@@ -0,0 +1,38 @@
+amends "../snippetTest.pkl"
+
+examples {
+  ["basic"] {
+    Bytes(List(1, 2, 3, 4))
+    Bytes("AQIDBA==")
+  }
+  ["equality"] {
+    Bytes(List()) == Bytes("")
+    Bytes(List(1, 2, 3, 4)) == Bytes("AQIDBA==")
+    Bytes(List(1, 2)) != Bytes(List(1, 2, 3, 4))
+  }
```

Comment:
```suggestion
  }
  ["length"] {
    Bytes(List()).length == 0
    Bytes(List(1, 2, 3, 4)).length == 4
    Bytes("AQIDBA==").length == 4
  }
```

Reply:
The following appears to work for me:
```pkl
  # Input
  ["subscript"] {
    Bytes(List(1, 2, 3, 4))[0] == 1
    Bytes(List(1, 2, 3, 4))[3] == 4
  }

  # Output
  ["subscript"] {
    true
    true
  }
```

Reply:
In addition to equality, should we also support comparison operations on a sequence of `Byte` objects? It is sometimes helpful to lexicographically sort bytes. Java has `Byte.compare` for this.